src/ConfigR/ObjectExtensions.cs

```diff
@@ -29,7 +29,7 @@ public static string ToJson(this object value)
             }
             catch (Exception ex)
             {
-                log.TraceFormat(CultureInfo.InvariantCulture, "Error converting '{0}' to JSON.", ex, value);
+                log.TraceFormat(string.Format(CultureInfo.InvariantCulture, "Error converting '{0}' to JSON.", ex));

```

Comment:
This is wrong. The original call passed `ex` as the exception argument and `value` as the formatting argument. I think LibLog has a similar overload. (This will all be simpler if you remove the `string.Format()` calls as per https://github.com/config-r/config-r/pull/205/files#r41540439.)